### DISCUSSION TITLE
Springboot 3 application request 404

### DISCUSSION THREAD

**Discussion by xiangtianyu** [open-telemetry/opentelemetry-java-instrumentation#9388] [Q&A] [answered]
I found my Springboot 3 application request is 404 when using otel agent

I tried to use different version of agent, and i found that when i used the version >= 1.19.0 it works well, but if the version < 1.18.0 all mappings in my app will return 404.

I tried to get the release note of the 1.19.0, and found nothing about it. Isthere any one can give me some information about this bug.

**Comment by mateuszrzeszutek** [comment]
Hey @xiangtianyu ,

Version 1.18.0 is nearly one year old, we don't provide support for versions that old. We recommend using the latest javaagent version.

**Reply by xiangtianyu** [reply]
Actually, i want to know what's the problem. And i will try to fix it in my otel agent version.

**Reply by mateuszrzeszutek** [reply, marked as answer]
The only thing that looks kinda suspicious in the release notes is #6766, you may want to try that one out.

**Reply by xiangtianyu** [reply]
Thanks, that's the problem.
